scripts/build.subr

```diff
@@ -537,6 +537,21 @@ buildworld () {
 # turn it back on afterwards.
 futurebuildworld () {
 	# Turn off ntpd if necessary
+	# Turn off salt too, so it won't turn ntp back on
```

Comment:
This is fine with me although I would suggest expanding the comment, because salt's not being used on the "official" FreeBSD-update build host.